resources/assets/js/jsvalidation.js

```diff
@@ -117,6 +117,10 @@ laravelValidation = {
             data = {};
             data[ element.name ] = value;
             data['_jsvalidation']= attribute;
+            var formMethod = $(validator.currentForm).attr('method');
+            if($(validator.currentForm).find('input[name="_method"]').length) {
+                formMethod = $(validator.currentForm).find('input[name="_method"]');
```

Comment:
@houndci, would you please have another look? Would be nice to see it merged, so I can take it over on my project.

Thanks!